### PR TITLE
Updating docs to add ShARC cfitsio

### PR DESCRIPTION
**Pull request by jkwmoore**
Mostly an update and rejig of the iceberg doc.

Edited and commented the install script.

### CODE REVIEW

File: sharc/software/libs/cfitsio.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.. _cfitsio:
```
Comment (willfurnass) [inline sharc/software/libs/cfitsio.rst:1]: rename label to _sharc_cfitsio or _cfitsio_sharc?

Comment (jkwmoore) [reply sharc/software/libs/cfitsio.rst]: Sorted!

**Comment by desmfryan** [comment]
Hi James

Looks fine.

The only comment I have is re install directory. I would not have included the /dev in the path i.e.
/usr/local/packages/libs/cfitsio/3.49/gcc-8.2

Cheers,

Des

**Comment by jkwmoore** [comment]
> 
> 
> Hi James
> 
> Looks fine.
> 
> The only comment I have is re install directory. I would not have included the /dev in the path i.e.
> /usr/local/packages/libs/cfitsio/3.49/gcc-8.2
> 
> Cheers,
> 
> Des

This has been amended in the files and on the cluster.

No idea what is going on with the spacing on the comments for the install script - tab spacing whackyness in Notepad++ ?

**Comment by jkwmoore** [comment]
Merging this now, promise to use 4 spaces instead of a tab in future - notepad++ from the UoS managed desktop a tab = 4 spaces, everyone else, ever, anywhere at any time - a tab = 8 spaces 

Smh :/

**Pull request merged**